src/plugin/sdl/sdl.c

```diff
@@ -99,10 +108,19 @@ static SDL_Texture *texture_buf;
 static SDL_Window *window;
 static ColorSpaceDesc SDL_csd;
 static Uint32 pixel_format;
+#if defined(HAVE_SDL2_TTF) && defined(HAVE_FONTCONFIG)
+static TTF_Font *sdl_font;
+static pthread_mutex_t sdl_font_mtx = PTHREAD_MUTEX_INITIALIZER;
+static SDL_RWops *sdl_font_rw;
+static int sdl_font_idx;
+static int sdl_font_size;
+static SDL_Color text_colors[16];
+static struct text_system Text_SDL;
+#endif
 static int font_width, font_height;
 static int win_width, win_height;
 static int m_x_res, m_y_res;
-static int use_bitmap_font;
+static int use_bitmap_font = 1;
```

Comment:
Why?

Reply:
There's no X based text rendering anymore, so if ttf isn't available rendering is vga.

Reply:
This is too pessimistic, you can't set up
the failure scenario right on start.
Please set up everything as if things going
to work, and change only on failures.

Reply:
Okay, will do.

Reply:
Now set on function return.